thirdparty/mupdf/external_fonts.patch

```diff
@@ -379,23 +404,25 @@ index be69ce1c..a24b2122 100644
  		const unsigned char *data;
  		int len;
  
-@@ -383,6 +437,15 @@ pdf_load_substitute_font(fz_context *ctx, pdf_font_desc *fontdesc, const char *f
+@@ -383,6 +437,17 @@ pdf_load_substitute_font(fz_context *ctx, pdf_font_desc *fontdesc, const char *f
  			fz_throw(ctx, FZ_ERROR_GENERIC, "cannot find substitute font");
  
  		fontdesc->font = fz_new_font_from_memory(ctx, fontname, data, len, 0, 1);
 +#else
-+		char * filename = pdf_lookup_substitute_font_from_file(ctx, mono, serif, bold, italic);
++		char *filename = pdf_lookup_substitute_font_from_file(ctx, mono, serif, bold, italic);
 +		if (!filename)
 +			fz_throw(ctx, FZ_ERROR_GENERIC, "cannot find substitute font");
-+
++	
```

Comment:
I feel like a total nitpicker for pointing out this spare tab but GH kinda highlights it in the diff. >_>

Reply:
I should turn on trailing whitespace trimming. :)